xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -963,8 +961,18 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
   {
     StringUtils::ToLower(strTmp);
     if (strTmp == "horizontal")
+    {
       orientation = HORIZONTAL;
+      // default orientation for a settings slider is vertical
```

Comment:
sure but no need to do so if this could easily be avoided. you won't break a skin for the sake of breaking it, right?